jekyll/_includes/ja/global-footer.html

```diff
@@ -0,0 +1,102 @@
+            <li><a href="https://ideas.circleci.com/" data-analytics-child='{ "menu": "feature-requests" }' target="_blank">改善要望</a></li>
+            <li><a href="{{ '/support/premium-support/' | outer_url }}" data-analytics-child='{ "menu": "premium-support" }'>プレミアム サポート</a></li>
+          </ul>
+        </div>
+        <div class="col-md-2 col-sm-3 col-xs-6">
+          <h6>関連資料</h6>
+          <ul class="list-unstyled">
+            <li><a href="{{ '/blog/' | outer_url }}" data-analytics-child='{ "menu": "blog" }'>ブログ</a></li>
+            <li><a href="{{ '/customers/' | outer_url }}" data-analytics-child='{ "menu": "case-studies" }'>ケース スタディー</a></li>
+            <li><a href="{{ '/resources/#white-papers' | outer_url }}" data-analytics-child='{ "menu": "white-papers" }'>ホワイト ペーパー</a></li>
+            <li><a href="{{ '/resources/events/' | outer_url }}" data-analytics-child='{ "menu": "events" }'>イベント</a></li>
+            <li><a href="{{ '/resources/#webinars' | outer_url }}" data-analytics-child='{ "menu": "webinars" }'>ウェビナー</a></li>
+            <li><a href="{{ '/resources/#videos' | outer_url }}" data-analytics-child='{ "menu": "videos" }'>ビデオ</a></li>
+          </ul>
+        </div>
+        <div class="col-md-2 col-sm-3 col-xs-6">
+          <h6>会社概要</h6>
+          <ul class="list-unstyled">
+            <li><a href="{{ '/about/' | outer_url }}" data-analytics-child='{ "menu": "about-us" }'>CircleCI について</a></li>
+            <li><a href="{{ '/careers/' | outer_url }}" data-analytics-child='{ "menu": "careers" }'>採用情報 <span class="text-blue">(メンバー募集中!)</span></a></li>
+            <li><a href="{{ '/about/team/' | outer_url }}" data-analytics-child='{ "menu": "team" }'>チーム紹介</a></li>
+            <li><a href="{{ '/contact/' | outer_url }}" data-analytics-child='{ "menu": "contact-us" }'>お問い合わせ</a></li>
+            <li><a href="{{ '/partners/' | outer_url }}" data-analytics-child='{ "menu": "partner-with-us" }'>パートナー募集</a></li>
+            <li><a href="{{ '/newsroom/' | outer_url }}" data-analytics-child='{ "menu": "newsroom" }'>メディア掲載</a></li>
+          </ul>
+        </div>
+        <div class="col-md-2 col-sm-3 col-xs-6">
+          <h6>CircleCI 入門</h6>
+          <ul>
+            <li><a href="{{ '/continuous-integration/' | outer_url }}" data-analytics-child='{ "menu": "what-is-ci" }'>CI とは?</a></li>
+            <li><a href="{{ '/2.0/' | language_relative_url }}" data-analytics-child='{ "menu": "how-to-get-started" }'>CircleCI の導入手順</a></li>
+          </ul>
+        </div>
+      </nav>
+    </div>
+    <div class="row margin-top-medium">
+      <div class="col-md-2 col-xs-4">
+        <img src="{{site.baseurl}}/assets/img/logos/logo-wordmark.svg" class="logo-small margin-top-small" alt="circleci">
+      </div>
+      <div class="col-md-4 col-md-offset-6 col-xs-offset-0 col-xs-8">
+        <div class="text-center-small-float-right">
+          {% include social-buttons.html %}
```

Comment:
I'm not seeing these social icons locally, but maybe that's just me, as they show up in Tyler's screenshot. The html elements are there, just not seeing them (maybe an issue with the icon font?)